### ISSUE TITLE
site support request: thechive.com

### ISSUE THREAD

**Issue by privong** [rg3/youtube-dl#5016]
```
$ youtube-dl --version
2015.02.19.1
```

```
$ youtube-dl http://thechive.com/2015/02/20/so-thats-what-a-set-of-redneck-bagpipes-sound-like-video/ --verbose
[debug] System config: []
[debug] User config: ['--console-title']
[debug] Command-line args: ['http://thechive.com/2015/02/20/so-thats-what-a-set-of-redneck-bagpipes-sound-like-video/', '--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.19.1
[debug] Python version 3.4.2 - Linux-3.18.6-1-ARCH-x86_64-with-arch
[debug] exe versions: ffmpeg 2.5.4, ffprobe 2.5.4, rtmpdump 2.4
[debug] Proxy map: {}
[generic] so-thats-what-a-set-of-redneck-bagpipes-sound-like-video: Requesting header
WARNING: Falling back on generic information extractor.
[generic] so-thats-what-a-set-of-redneck-bagpipes-sound-like-video: Downloading webpage
[generic] so-thats-what-a-set-of-redneck-bagpipes-sound-like-video: Extracting information
ERROR: Unsupported URL: http://thechive.com/2015/02/20/so-thats-what-a-set-of-redneck-bagpipes-sound-like-video/
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/youtube_dl/YoutubeDL.py", line 643, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3.4/site-packages/youtube_dl/extractor/common.py", line 269, in extract
    return self._real_extract(url)
  File "/usr/lib/python3.4/site-packages/youtube_dl/extractor/generic.py", line 1169, in _real_extract
    raise UnsupportedError(url)
youtube_dl.utils.UnsupportedError: Unsupported URL: http://thechive.com/2015/02/20/so-thats-what-a-set-of-redneck-bagpipes-sound-like-video/
```

**Comment by naglis** [comment]
Support for Kaltura embeds (which thechive.com uses) should be available in the next release of youtube-dl. Thanks for the report!

**Comment by privong** [comment]
Great, thanks @naglis!

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.02.28. See [our FAQ](https://yt-dl.org/update) if you need help updating.
